R/article.R

```diff
@@ -281,19 +281,67 @@ mnras_article <- function(..., keep_tex = TRUE, fig_caption = TRUE) {
   )
 }
 
+#' @param document_style For \code{oup_article}, one of "contemporary", "modern", or "traditional" setting overall style of document. Defaults to "contemporary".
+#' @param papersize For \code{oup_article}, one of "large", "medium", or "small" setting output page size. Defaults to "large".
+#' @param namedate For \code{oup_article}, a logical variable indicating if natbib citations should be in name-date format. Defaults to \code{FALSE}.
+#' @param onecolumn For \code{oup_article}, a logical variable indicating if one column formatting should be used. Defaults to \code{FALSE}.
 #' @section \code{oup_article}: Format for creating submissions to many Oxford University Press
 #'   journals. Adapted from
 #'   \url{https://academic.oup.com/journals/pages/authors/preparing_your_manuscript}
-#'   and \url{https://academic.oup.com/icesjms/pages/General_Instructions}.
+#'   and the \code{oup-authoring-template} available on CTAN at
+#'   \url{https://www.ctan.org/pkg/oup-authoring-template}.
 #' @export
 #' @rdname article
 oup_article <- function(
   ..., keep_tex = TRUE,
-  md_extensions = c("-autolink_bare_uris")
+  md_extensions = c("-autolink_bare_uris"),
+  journal=NULL,pandoc_args=NULL,
+  number_sections=TRUE,
+  citation_package=c("natbib","default"),
```

Comment:
OK, will do.

Reply:
I think I misspoke here. The new format will not work with `biblatex`, which is why I use match.arg on it and limit it to these two options. I will fix shortly to return the code to its original format.

Reply:
OK it was my understanding and wanted to confirm. It would be good to add a note in the help page for this function about biblatex not being supported.

Reply:
I have cleared this up in the documentation of the format function on the `newoldouptemplate` branch associated with PR #431.

Une chose rien à voir: Vous n'habitez pas à Montpellier par hasard ?